lib/pure/algorithm.nim

```diff
@@ -275,6 +283,15 @@ proc sorted*[T](a: openArray[T], cmp: proc(x, y: T): int {.closure.},
     result[i] = a[i]
   sort(result, cmp, order)
 
+proc sorted*[T](a: openArray[T], order = SortOrder.Ascending): seq[T] = 
+  ## Overload of ``sorted`` using ``system.cmp``
```

Comment:
Idem, description should say what the proc is doing, not introduce a layer of description indirection.

The signature can be a func/noSideEffect as well